lightning/src/chain/channelmonitor.rs

```diff
@@ -2008,33 +2008,37 @@ impl<Signer: Sign> ChannelMonitorImpl<Signer> {
 			self.is_paying_spendable_output(&tx, height, &logger);
 		}
 
-		self.block_confirmed(height, txn_matched, watch_outputs, claimable_outpoints, broadcaster, fee_estimator, logger)
+		self.block_confirmed(height, txn_matched, watch_outputs, claimable_outpoints, &broadcaster, &fee_estimator, &logger)
 	}
 
+	/// Update state for new block(s)/transaction(s) confirmed. Note that the caller *must* update
+	/// `self.best_block` before calling, and set `conf_height` to the height at which any new
+	/// transaction(s)/block(s) were confirmed at, even if it is not the current best height
+	/// (assuming we are being called with new transaction(s) confirmed).
```

Comment:
The parenthetical seems unnecessary and a little hard to follow. I'd suggest dropping it.